docs/cedana-image-streamer/cedana-image-streamer.md

```diff
@@ -0,0 +1,52 @@
+## Checkpoint/Restore with cedana-image-streamer
```

Comment:
This is great. Would separate into two sections, one for a CPU workload and another for a GPU workload